packages/jest-runtime/src/script_transformer.js

```diff
@@ -315,6 +316,36 @@ export default class ScriptTransformer {
         e.stack = e.codeFrame;
       }
 
+      if (
```

Comment:
can we also take it outside of this module and make it more declarative? it makes it really bloated :)
like
```js
if (
  e instanceof SyntaxError &&
  e.message.includes('Unexpected token') &&
  !e.message.includes(' expected')
) {
  e = addTransformErrorMessage(e);
}
```